test/spec/components/ui/markers_spec.coffee

```diff
@@ -87,6 +92,57 @@ define [], () ->
           it "is blank when free", ->
             expect(@component.iconBasedOnType(@component.attr.mapPinShadow, { free: true })).toEqual({ url: "" })
 
+    describe '#createMarker', ->
+      describe 'without label options', ->
+        beforeEach ->
+          @setupComponent()
+          @marker = @component.createMarker(datum)
+
+        it 'is a standard marker type', ->
+          expect(@marker.constructor).toEqual google.maps.Marker
+
+      describe 'with label options', ->
+        beforeEach ->
+          @setupComponent
+            markerLabelOptions: (datum) ->
+              content: datum.price
+          @marker = @component.createMarker(datum)
+
+        it 'is a standard marker type', ->
```

Comment:
Good catch. I updated the copy. The assertion is correct, but not the description.